### ISSUE TITLE
视频一到最后一秒就报错

### ISSUE THREAD

**Issue by bangzhu233** [YaoFANGUK/video-subtitle-remover#53]
![589AAA4048E8DAE40152EC0C9022283B](https://github.com/YaoFANGUK/video-subtitle-remover/assets/26674151/74c7673c-6fae-45f7-a12d-81871c0cee12)
![Uploading {CCACEF3A-326A-479a-AB00-07189226F201}.png…]()

**Comment by jonas87** [comment]
我犯了同样的错误

**Comment by Jongy36** [comment]
Hello YaoFANGUK,

Many thanks for all of your work.

This is precisely what I need but when I run the program, right at the end, this error message always appears (_please see screenshots_) I see that the application is successfully removing the subtitles but no new video file is created.

I have attempted this a number of times and re-downloaded the file as well as downloading the older version without success.

Then, the next day, I am no longer able to get as far as opening the GUI as an error message appears in Chinese. As I am unable to extract Chinese text, I do not know what the message means.

Would you be able to help with these difficulties?

By the way, I am not an IT professional. If you respond in Chinese, I can have your text translated.

Many thanks.

With best regards


![Clipboard_04-14-2024_01](https://github.com/YaoFANGUK/video-subtitle-remover/assets/167550828/8a2b0bce-1a62-49be-8b94-f47771c17dec)
![Clipboard_04-14-2024_02](https://github.com/YaoFANGUK/video-subtitle-remover/assets/167550828/011fa003-9aae-4c9d-8d5f-3be821f3e899)
![Error Message Chinese](https://github.com/YaoFANGUK/video-subtitle-remover/assets/167550828/ea03222f-d18c-4027-bbcb-76a09df1d3c8)

**Comment by YoungWiller** [comment]
有解决方案了吗

**Comment by Jongy36** [comment]
No. I have no solution. I am still waiting for a response from YaoFANGUK from whom I have heard nothing.

不，我没有解决办法。我仍在等待 天涯古巷 的答复，而我却一无所获。

**Comment by ggbond2077** [comment]
![image](https://github.com/user-attachments/assets/4b28b6fd-9b5d-4e0c-951a-e85062f2aace)
The cv2 get video frame count is not accurate. That's the root cause of this problem.
